extensions/commands/art/cmd_build_info.py

```diff
@@ -437,12 +442,12 @@ def build_info_upload(conan_api: ConanAPI, parser, subparser, *args):
         request_url = f"{request_url}?project={args.project}"
     response = api_request("put", request_url, user, password, json_data=json.dumps(build_info_json))
     if response:
-        cli_out_write(response)
+        return response
     else:
-        cli_out_write("Build info uploaded successfully.")
+        return "Build info uploaded successfully."
```

Comment:
This is weird as a result, it seems more like a ``ConanOutput()`` informational message